redhat-developer/app-services-cli
# feat(consumergroup): add consumergroup cmd with list subcommand

### Description

![Screenshot from 2021-04-05 17-52-31](https://user-images.githubusercontent.com/23582438/113573417-c062cc00-9637-11eb-8506-f522a40e9502.png)


Fixes #384 

### Verification Steps
1. Run `make mock-api/start` to start the mock API.
2. Run `rhoas login --api-gateway=http://localhost:8000` to use the mock API.
3. Run `rhoas kafka use somekafka` to use a Kafka instance (this will always pick the same one, it is not dynamic).
4. Run `go run ./cmd/rhoas kafka consumergroup list` to see all the consumer groups.

### Type of change


### Checklist
